### PR TITLE
update category page

### PR DESCRIPTION
**Pull request by ranitasahaa1230**
Added details of the quiz
Added error message

### CODE REVIEW

File: src/components/Category/Category.js [hunk 1]
```diff
@@ -1,7 +1,62 @@
...
+          onChange={(e) => setNames(e.target.value)}/>
+        <TextField
+          select
+          label="Select Category"
+          value={categories}
+          onChange={(e) => setCategories(e.target.value)}
+          variant="outlined"
+        >
+          {Categories.map((type) => (
+            <MenuItem key={type.category} value={type.value}>
```
Comment (anupkgurung) [inline src/components/Category/Category.js:40]: type can be destructured.

**Review by anupkgurung — APPROVED** [review]
data separation is nice but it should not be in components as it's not try to put it into a separate folder like constant or data asset etc

**Review by maitrakhatri — APPROVED** [review]
Code is clean and very easy to read.
I discovered the `variant` attribute/prop in react component.

File: src/App.css [hunk 2]
```diff
@@ -12,7 +12,7 @@
 }
 
 html {
-  font-size: 62.5%;
+  /* font-size: 62.5%; */
```
Comment (Veekshith-Thoram) [inline src/App.css:15]: Remove this comment

File: src/components/Categories/Categories.css [hunk 3]
```diff
@@ -1,5 +1,5 @@
-main-quiz{
-    margin-top: 12rem;
+.main-quiz{
+    margin-top: 2rem;
```
Comment (Veekshith-Thoram) [inline src/components/Categories/Categories.css:3]: Remove space here

**Review by Veekshith-Thoram — APPROVED** [review]

**Pull request merged**